compiler/luci/pass/src/FusePreActivationBatchNormPass.test.cpp

```diff
@@ -0,0 +1,368 @@
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "FusePreActivationBatchNormPassInternal.h"
+
+#include <luci/IR/CircleNodes.h>
+
+#include <math.h>
+#include <vector>
+
+#include <gtest/gtest.h>
+
+namespace
+{
+
+/**
+ *  Simple graph for test
+ *
+ *  BEFORE
+ *
+ *   [Conv] W + bias
+ *        \     [Conv]
+ *         \     /
+ *          [Add]
+ *         /    \
+ *        /    [Mul]   gamma
+ *       |       |
+ *       |   [Add+Relu]   beta
+ *       |       |
+ *       |     [Conv]   W + bias
+ *        \     /
+ *         [Add]
+ *
+ *  AFTER
+ *
+ *   [Conv] W + (bias + beta/gamma)
+ *        \     [Conv]
+ *         \     /
+ *          [Add]
+ *         /    \
+ *       |     [Relu]
+ *       |       |
+ *       |     [Conv]  (gamma * W) + (bias - beta/gamma)
+ *        \     /
+ *         [Add]
+ *
+ */
+class SimpleGraph
+{
+public:
+  SimpleGraph()
+  {
+    pred_conv = g.nodes()->create<luci::CircleConv2D>();
+    pred_conv_filter = g.nodes()->create<luci::CircleConst>();
+    pred_conv_bias = g.nodes()->create<luci::CircleConst>();
+    pred_conv2 = g.nodes()->create<luci::CircleConv2D>();
+    pred_add = g.nodes()->create<luci::CircleAdd>();
+    mul = g.nodes()->create<luci::CircleMul>();
+    mul_gamma = g.nodes()->create<luci::CircleConst>();
+    add = g.nodes()->create<luci::CircleAdd>();
+    add_beta = g.nodes()->create<luci::CircleConst>();
+    conv = g.nodes()->create<luci::CircleConv2D>();
+    conv_filter = g.nodes()->create<luci::CircleConst>();
+    conv_bias = g.nodes()->create<luci::CircleConst>();
+    succ_add = g.nodes()->create<luci::CircleAdd>();
+
+    pred_conv->dtype(loco::DataType::FLOAT32);
+    pred_conv_filter->dtype(loco::DataType::FLOAT32);
+    pred_conv_bias->dtype(loco::DataType::FLOAT32);
+    pred_conv2->dtype(loco::DataType::FLOAT32);
+    pred_add->dtype(loco::DataType::FLOAT32);
+    mul->dtype(loco::DataType::FLOAT32);
+    mul_gamma->dtype(loco::DataType::FLOAT32);
+    add->dtype(loco::DataType::FLOAT32);
+    add->fusedActivationFunction(luci::FusedActFunc::RELU);
+    add_beta->dtype(loco::DataType::FLOAT32);
+    conv->dtype(loco::DataType::FLOAT32);
+    conv_filter->dtype(loco::DataType::FLOAT32);
+    conv_bias->dtype(loco::DataType::FLOAT32);
+    succ_add->dtype(loco::DataType::FLOAT32);
+
+    pred_conv->shape({1, 12, 12, 64});
+    pred_conv_filter->shape({64, 1, 1, 64});
+    pred_conv_bias->shape({64});
+    pred_conv2->shape({1, 12, 12, 64});
+    pred_add->shape({1, 12, 12, 64});
+    mul->shape({1, 12, 12, 64});
+    mul_gamma->shape({64});
+    add->shape({1, 12, 12, 64});
+    add_beta->shape({64});
+    conv->shape({1, 12, 12, 64});
+    conv_filter->shape({64, 1, 1, 64});
+    conv_bias->shape({64});
+    succ_add->shape({1, 12, 12, 64});
```

Comment:
same reason like https://github.com/Samsung/ONE/pull/4858#discussion_r515717709,
if possible, please use small numbers for all above shapes

Reply:
I've reduced the dimensions.